pkg/bucketclass/reconciler.go

```diff
@@ -297,3 +298,50 @@ func (r *Reconciler) FinalizeDeletion() error {
 	}
 	return nil
 }
+
+// UpdateBucketClass updates all buckets that are assigned to a BucketClass
+func (r *Reconciler) UpdateBucketClass() error {
+	log := r.Logger
+
+	if r.BucketClass == nil {
+		return fmt.Errorf("BucketClass not loaded %#v", r)
+	}
+
+	policyTiers := []nb.TierItem{}
+	tiers := []nb.TierInfo{}
+
+	for i := range r.BucketClass.Spec.PlacementPolicy.Tiers {
+		tier := &r.BucketClass.Spec.PlacementPolicy.Tiers[i]
+		// Tier is irrelevant and will be populated in the BE
+		policyTiers = append(policyTiers, nb.TierItem{Order: int64(i), Tier: "SLOTH"})
```

Comment:
Tier SLOTH ? :)

Reply:
Any suggestions for a different name? It is irrelevant since I populate that in the BE :)

Reply:
TEMP ?